### ISSUE TITLE
Program doesn't run ValueError: max() arg is an empty sequence

### ISSUE THREAD

**Issue by yukaric317** [IIInitiationnn/BloodEmporium#8]
[debug-22-11-15 06-55-38.log](https://github.com/IIInitiationnn/BloodEmporium/files/10012508/debug-22-11-15.06-55-38.log)
Can't run the program

Here is the log 

initialisation, merging
capturing screen
matched origin: origin_basic_black.png
vector: circles and match to unlockables
hough transform: lines
creating networkx graph
NODES
    ORIGIN
EDGES
Traceback (most recent call last):
  File "state.py", line 180, in run
ValueError: max() arg is an empty sequence

**Comment by IIInitiationnn** [comment]
Seems like the app isn't detecting any of the items on the bloodweb. Please check your resolution and UI scale are correct. If they are, please send a screenshot or video, as well as your `config.json` file so I can have a look.

**Comment by tutzeroone** [comment]
@IIInitiationnn sorry, I'm hijacking this, since OP hasn't answered and I'm having the same issue.

I have a 1920*1080 display, the game run in regular full screen mode and is set to 100% UI scale ; so is BloodEmporium. I haven't changed any settings. Config is joined (extension changed for github).

Thanks for your help and work on the project !

[config.json.TXT](https://github.com/IIInitiationnn/BloodEmporium/files/10087097/config.json.TXT)

**Comment by IIInitiationnn** [comment]
So far for all the people who have had this issue, it seems to have been related to having multiple monitors and not using the correct one. I'll provide a list of things you can check, and you can let me know if any of them fix the issue, if not I'll probably need a little more information.

1. Correctly set your resolution and UI scale.
2. If you are using a custom icon pack, set your path to the correct installation folder (if you have an installation which is not at the default location).
3. Ensure the path to Blood Emporium.exe does not contain any special unicode characters - e.g. some users had Russian and Chinese characters in a parent folder of the executable, causing it to bug out - see https://github.com/IIInitiationnn/BloodEmporium/issues/5.
4. Disable any shaders / visual filters.
5. Try swapping your game onto every monitor (if you have multiple).

**Comment by tutzeroone** [comment]
Hello and thanks for your quick answer !

I followed your instructions and it worked but I'm unsure which exact setting(s) did it, my guess is it was probably an issue with image recognition.

1/2/3/5. The resolution and scale were already set to 100%, both in game and in Blood Emporium, the game was not modified in any way (from Steam), the path was already using only ASCII characters and I have a single monitor.

4. The graphics settings were already set to "LOW" and I was unsure about how to disable any shaders / visual filters in the game so I've manually turned off any form of optimization in the control panel of my GPU as in the picture below (picture for future reference)

![dbd](https://user-images.githubusercontent.com/119010991/204057481-1cd646d3-9dce-4949-82e4-c3823ecc19da.png)

Thanks again !
